api/v1alpha1/drcluster_types.go

```diff
@@ -1,18 +1,6 @@
-/*
-Copyright 2022 The RamenDR authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-	http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+// Copyright 2022 The RamenDR authors.
+// SPDX-FileCopyrightText: The RamenDR authors
```

Comment:
The reuse tool keeps the existing copyright lines, but we don't really needed them
since they are replaced by the SPDX-FileCopyrightText headers. I kept them in case
we want to keep them.

Reply:
@ShyamsundarR?

Reply:
Removed in current version in a separate commit to make it easy to revert:
https://github.com/RamenDR/ramen/pull/571/commits/1497e0dcfe9e1d6315c53997183a2b4173064766

Reply:
I squashed the removal into the first commit in the current version.